src/components/CavaticaProjectList/CavaticaProjectItem.js

```diff
@@ -45,17 +46,71 @@ const StudyLink = ({study, hideLink}) => {
   }
 };
 
+const UnlinkButton = ({unlinkProject, study, projectId}) => {
+  if (unlinkProject && study) {
```

Comment:
It's probably a better idea to move the conditional rendering here into the parent. There's no need to include the component if it's not going to be displayed.

Reply:
You may want to add proptypes for this component if you're concerned about what values you're getting for it.